sale_operating_unit/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===============================
+Operating Unit in Sales
+===============================
+
+This module was written to extend the Sales capabilities of Odoo.
+This module introduces the operating unit to the Sales Order.
+Security rules are defined to ensure that users can only display the
+Sales Orders in which they are allowed access to.
+
+Installation
+============
+
+No additional installation instructions are required.
+
+Configuration
+=============
+
+Go to 'Settings / Technical / Actions / User-defined Defaults' and remove
+the default set for the Shop.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/213/7.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/operating-unit/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Eficent <contact@eficent.com>
```

Comment:
Change to the actual contributors:
* Jordi Ballester Alomar <jordi.ballester@eficent.com>
* Aarón Henríquez <aheficent@eficent.com>
* Sudhir Arya <sudhir.arya@serpentcs.com>